etna/models/linear.py

```diff
@@ -64,6 +84,16 @@ def __init__(self, fit_intercept: bool = True, **kwargs):
             base_model=_LinearAdapter(regressor=LinearRegression(fit_intercept=self.fit_intercept, **self.kwargs))
         )
 
+    def params_to_tune(self) -> Dict[str, "BaseDistribution"]:
```

Comment:
Should it be the adapter's method or it is method of the model?

Reply:
In catboost for instance it is private in adapter and prophet adapter doesn't have it at all

Reply:
It is decided to be done not using adapter method because we have the same adapter for LinearModel and ElasticNet, but different grids.